sdk/nodejs/src/request.ts

```diff
@@ -1,11 +1,21 @@
 import axios, { AxiosRequestConfig, AxiosRequestHeaders, AxiosResponse } from 'axios'
+import axiosRetry from 'axios-retry'
 import { SDKEventBatchRequestBody, DVCLogger } from '@devcycle/types'
 import { DVCPopulatedUser } from './models/populatedUser'
 import { DVCEvent, DVCOptions } from './types'
 
 const axiosClient = axios.create({
+    timeout: 60 * 1000,
```

Comment:
this should be like 2 seconds IMO

Reply:
I guess I should pass that timeout through to this.

Reply:
yea its already passed through, changing the default to 10 sec? 
<img width="433" alt="image" src="https://user-images.githubusercontent.com/1219069/195667677-8edb8afd-6644-4144-8e78-2a64570ccfc7.png">